tests/slurm-tests/slurm_test_llama_nemotron_super_49B_v1._5_evals/check.py

```diff
@@ -0,0 +1,338 @@
+import argparse
+import io
+import json
+import os
+import sys
+
+# ---------------- Core (reasoning_on/off) ----------------
+REASONING_BENCH = ["math-500", "aime24", "aime25", "gpqa", "mmlu-pro", "livecodebench", "scicode", "hle"]
```

Comment:
I think it would be a good idea to split the benchmark constants, their performance, and the code that verifies the performance into separate files. In future we may want to add more/different models to check. The separation of verification logic from the data would allow for quick integration then.